src/templates/Category/query.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable filenames/match-exported */
+import React from 'react';
+
+import { BaseQuery, IBaseQuery } from '../../baseQuery';
+
+export type IPost = Omit<IBaseQuery, 'fileAbsolutePath'> & {
+  FeaturedImage: React.FC;
+};
+
+export interface ICategoryContext {
+  category: string;
+  newerPostsPath?: string;
+  olderPostsPath?: string;
+  posts: IPost[];
+  skip: string;
+}
+
+const query: QueryFn<IBaseQuery & ICategoryContext> = (
+  graphql: GraphqlFn
+) => graphql<IBaseQuery & ICategoryContext>(`
+  ${BaseQuery}
+
+  query CategoryTemplateQuery($category: String) {
```

Comment:
@kevcenteno I can't quite get this query to work properly. As of right now if you visit any of the category URLs (e.g. http://localhost:8000/category/online-fraud-prevention), it still shows all posts and not the ones filtered by the category.

Reply:
@kevcenteno I'm unfortunately still stuck on this. I tried a different approach of updating the `CreateMdxPages.ts` file accordingly: https://gist.github.com/dhogan8/ef9dbd8e4e6c3a6fec56d7add570c55d#file-createmdxpages-ts-L40-L44.

But while it shows the filtered for `category-ip-geolocation`, it shows the same exact posts for the `online-fruad-prevention`. In other words, only posts with the geolocation category are showing up on the /category/online-fraud-prevention page.